java/src/main/resources/local/application.yml

```diff
@@ -1,6 +1,13 @@
 spring:
+  profiles:
+    include:
+      - aws
+      - azure
+      - cosmosdb
+      - servicebus
+
   application:
-    name: stacks-api-cqrs
+    name: com.amido.stacks-api-cqrs-events
```

Comment:
Same question about the name change - why?

Reply:
fixed